<ui version="4.0" >
 <class>loadTest</class>
 <widget class="QDialog" name="loadTest" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Load testing</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Keys list</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QListWidget" name="keysList" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>(0 means random value)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="key" />
     </item>
     <item>
      <widget class="QToolButton" name="addKey" >
       <property name="text" >
        <string>+</string>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="clearList" >
     <property name="text" >
      <string>Clear list</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Send time (ms)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sendTimer" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>100000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="start" >
       <property name="text" >
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop" >
       <property name="text" >
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>loadTest</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>268</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearList</sender>
   <signal>clicked()</signal>
   <receiver>keysList</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>227</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel" >
     <x>137</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
